.travis.yml

```diff
@@ -26,48 +26,51 @@ env:
     - MAIN_GO_VERSION=1.12
     - GORACE="halt_on_error=1"
     - FORCE_TM_TEST=1
-
-install:
-- make deps
-- go get -d github.com/tendermint/tendermint/... ;
-  cd ${GOPATH}/src/github.com/tendermint/tendermint ;
-  git checkout ${TM_VERSION} ;
-  make get_tools && make check && make install ;
-  cd -;
+#
+#install:
+#- make deps
+#- go get -d github.com/tendermint/tendermint/... ;
+#  cd ${GOPATH}/src/github.com/tendermint/tendermint ;
+#  git checkout ${TM_VERSION} ;
+#  make get_tools && make check && make install ;
+#  cd -;
 
 script:
-- make lint;
-- make test;
-- if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" ]]; then
-     make cover;
-     ./coverage/upload.sh;
-   fi;
-- if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" && "$TRAVIS_OS_NAME" == "linux" ]]; then
-    release_latest=$( [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_TAG" == "" && "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]] && echo "yes" || echo "no" );
-    release_tag=$( [[ "$TRAVIS_TAG" != "" ]] && echo "yes" || echo "no" );
-
-    if [[ $release_latest == "yes" || $release_tag == "yes" ]]; then
-      make dist;
-    fi
-
-    if [[ $release_latest == "yes" ]]; then
-      docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-      docker tag  "iov1/bcpd:${BUILD_VERSION}" "iov1/bcpd:latest" ;
-      docker push "iov1/bcpd:latest";
-      docker tag  "iov1/bnsd:${BUILD_VERSION}" "iov1/bnsd:latest" ;
-      docker push "iov1/bnsd:latest";
-      docker logout;
-    fi;
-
-    if [[ $release_tag == "yes" ]]; then
-      docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-      docker tag  "iov1/bcpd:${BUILD_VERSION}" "iov1/bcpd:$TRAVIS_TAG" ;
-      docker push "iov1/bcpd:$TRAVIS_TAG";
-      docker tag  "iov1/bnsd:${BUILD_VERSION}" "iov1/bnsd:$TRAVIS_TAG" ;
-      docker push "iov1/bnsd:$TRAVIS_TAG";
-      docker logout;
-    fi;
-  fi;
+#- make lint;
+#- make test;
+#- if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" ]]; then
+#     make cover;
+#     ./coverage/upload.sh;
+#   fi;
+- printenv | grep -Ev "PATH|path|GEM" > env.list
+- docker run -it --rm --env-file env.list -v "$(pwd):/usr/src/app" iov1ops/danger:latest > danger.out; cat danger.out
+- grep -q 'Errors' danger.out && exit 1
+#- if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" && "$TRAVIS_OS_NAME" == "linux" ]]; then
```

Comment:
Huh? Why is this all commented out?

I guess just for testing. I saw review requested, but obviously not done. Ping for a proper review when you think it is finished